--- a/libavcodec/videotoolbox.c
+++ b/libavcodec/videotoolbox.c
@@ -845,8 +845,7 @@
                                                                    &kCFTypeDictionaryValueCallBacks);
 
     CFDictionarySetValue(config_info,
-                        avctx->codec_id == AV_CODEC_ID_HEVC
-                        || avctx->codec_id == AV_CODEC_ID_PRORES
+                        avctx->codec_id == AV_CODEC_ID_PRORES
                         || avctx->codec_id == AV_CODEC_ID_PRORES_RAW
                             ?
                             kVTVideoDecoderSpecification_EnableHardwareAcceleratedVideoDecoder :
